pkg/inject/report.go

```diff
@@ -144,19 +147,19 @@ func (r *Report) disableByAnnotation(conf *ResourceConfig) (bool, string) {
 	nsAnnotation := conf.nsAnnotations[k8s.ProxyInjectAnnotation]
 
 	if conf.origin == OriginCLI {
-		return podAnnotation == k8s.ProxyInjectDisabled, ""
+		return podAnnotation == k8s.ProxyInjectDisabled, "", ""
 	}
 
 	if nsAnnotation == k8s.ProxyInjectEnabled {
 		if podAnnotation == k8s.ProxyInjectDisabled {
-			return true, injectDisableAnnotationPresent
+			return true, injectDisableAnnotationPresent, annotationAtNamespace
```

Comment:
In this case the annotation at the workload level overrode the annotation at the namespace level, so it should be `annotationAtWorkload`

Reply:
Okay. Makes sense.